jupyter_book/book_template/content/guide/04_faq.md

```diff
@@ -78,3 +78,33 @@ Note that, if viewing the file on a Jupyter Notebook session, the figure will no
 
 If you've got questions, concerns, or suggestions, please open an issue at
 [at the jupyter book issues page](https://github.com/jupyter/jupyter-book/issues)
+
+## How should I add cell tags to my notebooks?
+
+You can control the behavior of Jupyter Book by putting custom tags
```

Comment:
Very cool !! Should we link out here to the sections on what kind of meta-data you might want to add ? Might motivate this a little more :)

Reply:
yeah that's a good idea!